bigbluebutton-config/bin/apply-lib.sh

```diff
@@ -44,9 +44,10 @@ fi
 
 HOST=$(cat $SERVLET_DIR/WEB-INF/classes/bigbluebutton.properties $tmpfile $BBB_WEB_ETC_CONFIG | grep -v '#' | sed -n '/^bigbluebutton.web.serverURL/{s/.*\///;p}' | tail -n 1)
 
-HTML5_CONFIG=/usr/share/meteor/bundle/programs/server/assets/app/config/settings.yml
-BBB_WEB_CONFIG=$SERVLET_DIR/WEB-INF/classes/bigbluebutton.properties
-
+HTML5_CONFIG=/etc/bigbluebutton/bbb-html5.yml
+if [ ! -f HTML5_CONFIG ]; then
```

Comment:
This is checking for a file named `HTML5_CONFIG`, not the filename from the variable.

Should be:

```suggestion
if [ ! -f "${HTML5_CONFIG}" ]; then
```